core/ui/ViewTemplate/body/global.tid

```diff
@@ -0,0 +1,24 @@
+title: $:/core/ui/ViewTemplate/body/global
+
+\procedure display-global()
+<div class="tc-global-tiddler-body">
+<div class="tc-global-tiddler-body-heading">
+<h2 class="tc-global-tiddler-body-type">
+<$let
+	name={{{ [<currentTiddler>removeprefix[$:/global/]] }}}	
+	type={{{ [{!!_is_macro}match[yes]then[macro]] :else[{!!_is_function}match[yes]then[function]] :else[{!!_is_procedure}match[yes]then[procedure]] :else[{!!_is_widget}match[yes]then[widget]] :else[[variable]] }}}
+>
+{{$:/language/Globals/Caption}}
```

Comment:
@Jermolene .. It would be nice, if the code would be indented to be more readable and imo `\whitespace trim` should be standard now

Reply:
Yes sorry about that, I'll attend to it.